service/src/test/java/uk/gov/hmcts/reform/fpl/service/ConfidentialDetailsServiceTest.java

```diff
@@ -461,7 +461,7 @@ void shouldReturnOthersIfOthersIsPrePopulated() {
         }
 
         @Test
-        void shouldPrepareOtherWithConfidentialValuesWhenConfidentialOthersIsNotEmpty() {
+        void shouldPrepareOtherWithConfidentialValuesRemoved() {
```

Comment:
This only tests the one 'other' can extend or have a new test to ensure multiple others have confidential fields persisted.

Reply:
done